sources/core/objects/Floor.cpp

```diff
@@ -28,7 +28,6 @@ void Floor::AfterWorldCreation()
 {
     ITurf::AfterWorldCreation();
     SetOpen(open_);
-    SetState(floor_type_);
```

Comment:
But SetOpen() is called in AfterWorldCreation and it works right for plating and for floors too (open_ = false -> SetState(floor_type_), true -> SetState("plating")).